packages/host/src/components/Footer/StoreSelector.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FC } from 'react';
+import {
+  Box,
+  CircularProgress,
+  FlatButton,
+  PaperPopoverSection,
+  useAuthContext,
+  usePaperClickPopover,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { StoreRowFragment, useStores } from '@openmsupply-client/system';
+
+export const StoreSelector: FC = ({ children }) => {
+  const { store, setStore } = useAuthContext();
+  const { hide, PaperClickPopover } = usePaperClickPopover();
+  const { data, isLoading } = useStores();
+  const t = useTranslation('app');
+
+  const storeSorter = (a: StoreRowFragment, b: StoreRowFragment) => {
```

Comment:
yes, duplicated from login. I still hope that the api will provide a sort option